src/pages/_projects-details.scss

```diff
@@ -2,151 +2,154 @@
-.projects-details .activity {
-  flex-direction: column;
-}
-
-.projects-details .project {
-  gap: 2.5rem;
-}
-
-.projects-details .project .header {
-  align-items: center;
-}
-
-.projects-details .project .mini-profile .title {
-  font-size: 1.25rem;
-}
-
-.projects-details .project .intro {
-  overflow-x: auto;
-}
-
-.projects-details .project .intro .lead,
-.projects-details .project .description .lead,
-.projects-details .project .checklist .lead {
-  margin-bottom: 0.8rem;
-}
+.projects-details {
+  .grid {
+    display: grid;
+    grid-template-columns: 350px 1fr 310px;
+  }
 
-.projects-details .stats {
-  gap: 5rem;
-}
+  .container {
+    padding: map-get(variables.$container-padding, "lg");
+    gap: 1rem;
+    overflow-x: hidden;
+  }
 
-.projects-details .project .checklist .progress-bar {
-  margin-bottom: 1rem;
-}
+  .related-projects-list {
+    background-color: map-get(variables.$colors, "bg");
+    position: sticky;
+    align-self: start;
+    border-right: 1px solid map-get(variables.$colors, "border");
+    top: app-header.$app-header-height;
+    overflow-y: auto;
+    height: calc(100vh - app-header.$app-header-height);
+
+    .searchbar {
+      background-color: map-get(variables.$colors, "bg");
+      border-bottom: 1px solid map-get(variables.$colors, "border");
+      position: sticky;
+      align-self: start;
+      top: 0;
+
+      input {
+        padding-top: map-get(variables.$form-controls-padding-y, "lg");
+        padding-bottom: map-get(variables.$form-controls-padding-y, "lg");
+        border: none;
+      }
+    }
+
+    .related-projects-list-item {
+      border-bottom: 1px solid map-get(variables.$colors, "border");
+    }
+  }
 
-.projects-details .project .checklist .tasks {
-  margin-bottom: 0.5rem;
-}
+  .project {
+    display: flex;
+    flex-direction: column;
+    gap: 2.5rem;
+
+    .header {
+      display: flex;
+      align-items: center;
+    }
+
+    .mini-profile .title {
+      font-size: 1.25rem;
+    }
+
+    .intro {
+      overflow-x: auto;
+
+      .lead {
+        margin-bottom: 0.8rem;
+      }
+    }
+
+    .description .lead {
+      margin-bottom: 0.8rem;
+    }
+
+    .checklist {
+      .lead {
+        margin-bottom: 0.8rem;
+
+        .tasks {
+          margin-bottom: 0.5rem;
+        }
+      }
+
+      .progress-bar {
+        margin-bottom: 1rem;
+      }
+
+      .add-task {
+        margin-left: auto;
+      }
+    }
+  }
 
-.projects-details .project .checklist .add-task {
-  margin-left: auto;
-}
+  .activity {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
 
-.projects-details .activity .tabs {
-  margin-left: map-get(variables.$container-padding, "lg") * -1;
-  margin-right: map-get(variables.$container-padding, "lg") * -1;
-  padding: 0 map-get(variables.$container-padding, "lg");
-}
+    .tabs {
+      margin-left: map-get(variables.$container-padding, "lg") * -1;
+      margin-right: map-get(variables.$container-padding, "lg") * -1;
```

Comment:
why `-1`?

Reply:
No, be more specific. What does `-1` do here? If not sure, find that stackoverflow question

Reply:
I wanted to use the negative value of the CSS variable and found the following solution:

The questing was "[How can I get a negative value of a CSS variables in a calc() expression?](https://stackoverflow.com/questions/48639882/how-can-i-get-a-negative-value-of-a-css-variables-in-a-calc-expression)".